Sources/Prometheus/Docs.docc/index.md

```diff
@@ -0,0 +1,19 @@
+# ``Prometheus``
+
+A prometheus client library for Swift.
+
+## Overview
```

Comment:
Can we add installation instructions to the index here as well. Allows us to point people directly at the documentation.

Reply:
Can we do this in a follow up?